rule/imports-blacklist.go

```diff
@@ -0,0 +1,74 @@
+package rule
+
+import (
+	"fmt"
+	"go/ast"
+	"strings"
+
+	"github.com/mgechev/revive/lint"
+)
+
+// ImportsBlacklistRule lints given else constructs.
+type ImportsBlacklistRule struct{}
+
+// Apply applies the rule to given file.
+func (r *ImportsBlacklistRule) Apply(file *lint.File, arguments lint.Arguments) []lint.Failure {
+	var failures []lint.Failure
+	blacklist := make([]string, len(arguments))
```

Comment:
We can store the blacklist as a set - `map[string]bool` so that we can have the algorithm in `Visit` below with `O(n)` instead of `O(n*m)`.

Reply:
A `map[string]struct{}` is also an alternative with a smaller footprint (https://play.golang.org/p/Mv1SgehJvbh)